inspirehep/bat/locators.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from selenium.webdriver.common.by import By
+
+
+# Literature Classes
+class LoginPageLocators(object):
+    LOG_OUT_BUTTON = (By.XPATH, '(//button[@type="button"])[2]')
+    SUBMIT_BUTTON = (By.XPATH, '//button[@type="submit"]')
+    PROFILE_BUTTON = (By.ID, 'user-info')
+    PASSW_TEXT_BOX = (By.ID, 'password')
+    EMAIL_TEXT_BOX = (By.ID, 'email')
+
+
+class CreateLiteraturePageLocators(object):
+    SUBMIT_BUTTON = (By.XPATH, '//div[@id="webdeposit_form_accordion"]/div[4]/span/button')
+    ACCEPT_BUTTON = (By.XPATH, '//button[@class="btn btn-warning"]')
+    SKIP_IMPORT_DATA = (By.ID, 'skipImportData')
+    LITERATURE_FORM = (By.ID, 'form_container')
+    DROPDOWN_BUTTON = (By.XPATH, '(//button[@type="button"])[8]')
+    DROPDOWN_CHECKBOX = (By.XPATH, 'input[type=\"checkbox\"]')
+    CONFERENCE_TAB = (By.LINK_TEXT, 'Conference Information')
+    PROCEEDINGS_TAB = (By.LINK_TEXT, 'Proceedings Information (if not published in a journal)')
+    REFERENCES_TAB = (By.LINK_TEXT, 'References')
+    COMMENTS_TAB = (By.LINK_TEXT, 'Additional comments')
+    ARXIV_ERROR_MSG = (By.ID, 'state-arxiv_id')
+    DOI_ERROR_MSG = (By.ID, 'state-doi')
+    URL_ERROR_MSG = (By.ID, 'state-url')
+    DATE_ERROR_MSG = (By.ID, 'state-thesis_date')
+
+
+class HoldingpenLiteratureListPageLocators(object):
+    RECORD_ENTRY = (By.XPATH, '//div[@class="row hp-item ng-scope"][1]')
+    SHOW_ABSTRACT_BUTTON = (By.XPATH, '//div[@class="row hp-item ng-scope"][1]/div/div/div[2]/holding-pen-template-handler/div[3]/a')
+
+
+class HoldingpenLiteratureDetailPageLocators(object):
+    PRINCIPAL_RECORD_PANEL = (By.XPATH, '(//div[@class="ng-scope"])[2]')
+    SUBMISSION_PANEL = (By.XPATH, '//p[@class="text-center ng-scope"]')
```

Comment:
No they are respectively:

- This panel
![image](https://cloud.githubusercontent.com/assets/4679712/25189723/4c6d8fce-252a-11e7-997b-27c8fdecddd2.png)

- And this panel 
![image](https://cloud.githubusercontent.com/assets/4679712/25189762/6ea0c48a-252a-11e7-84d7-3e6cdc1181a5.png)